sdk/src/main/java/io/radar/sdk/RadarApiHelper.kt

```diff
@@ -114,6 +114,7 @@ internal open class RadarApiHelper(
                 urlConnection.disconnect()
             } catch (e: IOException) {
                 handler.post {
+                    logger?.e("Received network error | error = $e");
```

Comment:
This should also be logged before the call to `handler.post`, since it has the potential to log in a separate thread, which is more optimal.